components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx

```diff
@@ -4,34 +4,71 @@
 import { StartWorkspaceModalKeyBinding } from "../App";
 import { LinkButton } from "@podkit/buttons/LinkButton";
 import { Heading2, Subheading } from "@podkit/typography/Headings";
+import { trackVideoClick } from "../Analytics";
+
+declare global {
+    interface Window {
+        onYouTubeIframeAPIReady: () => void;
+        YT: any;
+    }
+}
 
 export const EmptyWorkspacesContent = () => {
+    useEffect(() => {
+        // Load the YouTube IFrame Player API code asynchronously
+        const tag = document.createElement("script");
+        tag.src = "https://www.youtube.com/iframe_api";
+        const firstScriptTag = document.getElementsByTagName("script")[0];
+        firstScriptTag.parentNode?.insertBefore(tag, firstScriptTag);
+
+        // Create YouTube player when API is ready
+        window.onYouTubeIframeAPIReady = () => {
+            new window.YT.Player("gitpod-video", {
+                events: {
+                    onStateChange: onPlayerStateChange,
+                },
+            });
+        };
+    }, []);
+
+    const onPlayerStateChange = (event: any) => {
+        if (event.data === window.YT.PlayerState.PLAYING) {
+            trackVideoClick("create-new-workspace");
+        }
+    };
+
     return (
         <div className="app-container flex flex-col space-y-2">
-            <div className="px-6 py-3 flex flex-col">
-                <div className="flex flex-col items-center justify-center h-96 w-96 mx-auto">
-                    <Heading2 className="text-center pb-3">No Workspaces</Heading2>
-                    <Subheading className="text-center pb-6">
-                        Prefix any Git repository URL with {window.location.host}/# or create a new workspace for a
-                        recently used project.{" "}
-                        <a
-                            className="gp-link"
-                            target="_blank"
-                            rel="noreferrer"
-                            href="https://www.gitpod.io/docs/getting-started/"
-                        >
-                            Learn more
-                        </a>
-                    </Subheading>
-                    <span>
-                        <LinkButton href={"/new"} className="gap-1.5">
-                            New Workspace{" "}
-                            <span className="opacity-60 hidden md:inline">{StartWorkspaceModalKeyBinding}</span>
-                        </LinkButton>
-                    </span>
+            <div className="px-6 py-3 flex flex-row items-center justify-center space-x-16">
+                <div>
+                    <iframe
+                        id="gitpod-video"
+                        width="535"
+                        height="307"
+                        src="https://www.youtube.com/embed/R6FQ39sitAQ?enablejsapi=1&modestbranding=1&rel=0&controls=1&showinfo=0&fs=1"
+                        title="YouTube - Gitpod in 120 seconds"
```

Comment:
@Siddhant-K-code if we remove this line, would it remove the title from the thumbnail of the video? Imo it's best to have this look as clean as possible, almost as if we had just embedded it ourselves. 

<img width="374" alt="image" src="https://github.com/gitpod-io/gitpod/assets/83109781/c1ca2c0a-4303-4038-bcd4-51a0b4ec6b0f">

Reply:
> if we remove this line, would it remove the title from the thumbnail of the video?

Nope, It's a YT Embed styles